src/table/types.ts

```diff
@@ -314,6 +316,9 @@ export interface AdjusterProps {
   column: Column;
   isLastColumn: boolean;
   onColumnResize?: () => void;
+  borderHeight?: number | undefined;
```

Comment:
You  don't need to set `number | undefined` when you set the property as optional